components/automate-deployment/cmd/deployment-service/inspect-db.go

```diff
@@ -152,10 +152,16 @@ func pretty(buf io.Writer, v reflect.Value, depth int) {
 
 		fmt.Fprint(buf, "{\n")
 		for i := 0; i < v.NumField(); i++ {
+			field := v.Field(i)
+			fieldName := v.Type().Field(i).Name
+			if skipPBInternal(fieldName, field) {
+				continue
+			}
+
 			fmt.Fprint(buf, nextIndent)
-			fmt.Fprint(buf, v.Type().Field(i).Name)
+			fmt.Fprint(buf, fieldName)
 			fmt.Fprint(buf, ": ")
-			pretty(buf, v.Field(i), depth+1)
+			pretty(buf, field, depth+1)
```

Comment:
Not a part of your PR but I'm curious: What's the difference between a field and a fieldName? Is it the value?

Reply:
Eh I see some examples below, nm